### ISSUE TITLE
Alignment doesn't work for org table with indic scripts

### ISSUE THREAD

**Issue by rnchzn** [casouri/valign#27]
Alignment doesn't work for org table with indic scripts.

Here is the unit test file.   The file has babel blocks that sets up the default fonts for scripts.  Please execute it before viewing the file.

[itrans-unicode-mapping-tamil.org.txt](https://github.com/casouri/valign/files/6986042/itrans-unicode-mapping-tamil.org.txt)

This is what I see with latest `valign` and  `emacs -q`.

![Screenshot from 2021-08-14 12-19-00](https://user-images.githubusercontent.com/82300681/129437858-b83759cf-53c0-43a2-aa36-abafd45c9035.png)

**Comment by rnchzn** [comment]
I just bumped the value of `valign-max-table-size` and it works now.  

As you note in the README, aligning above table  takes a long time.  

Meanwhile, I have some prototype code that uses Org's official parser to prettify above table.  It seems faster than your implementation.  **Please leave the bug open, I will cleanup my prototype code, and share it with you.**

TLDR:  Have you experimented with Org's defautlt parser for speeding up things?

**Comment by casouri** [comment]
I see, thanks for experimenting with this. I'll have a look at their parser.
